src/platform/forms/save-in-progress/RoutedSavableApp.jsx

```diff
@@ -20,6 +20,8 @@ import { getSaveInProgressState } from './selectors';
 import environment from 'platform/utilities/environment';
 import { APP_TYPE_DEFAULT } from '../../forms-system/src/js/constants';
 
+import { restartShouldRedirect } from 'applications/static-pages/wizard';
```

Comment:
Same story here. To eliminate the application dependency from platform code, put this function in another module which _doesn't_ import application code, and then import it from there.

Reply:
Now in `platform/site-wide/wizard`